src/test/java/net/serenitybdd/starter/fetests/pageObjects/InventoryPage.java

```diff
@@ -0,0 +1,28 @@
+package net.serenitybdd.starter.fetests.pageObjects;
+
+import lombok.Getter;
+import net.serenitybdd.core.annotations.findby.FindBy;
+import net.serenitybdd.core.annotations.findby.How;
+import net.serenitybdd.core.pages.WebElementFacade;
+import net.thucydides.core.annotations.DefaultUrl;
+import net.thucydides.core.annotations.NamedUrl;
+import net.thucydides.core.annotations.NamedUrls;
+import net.thucydides.core.annotations.Step;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Getter
+@NamedUrls({
+        @NamedUrl(name = "inventory", url = "/inventory.html")
+})
+public class InventoryPage extends BasePage {
+
+    @FindBy(how = How.XPATH, using = "//div[@class='header_secondary_container']/span[text()='Products']")
+    private WebElementFacade inventoryPageHeader;
+
+    @Step("Check if inventory page is loaded")
```

Comment:
I recommend to split Page and Step layers. You'll get benefits when scaling your Tests and TAF